z2/resources/chain-specs/zq2-devnet.toml

```diff
@@ -1,5 +1,5 @@
 p2p_port = 3333
-bootstrap_address = [ "12D3KooWJNUNgs1UM9mw1Pfskxk4o3VWz37cpneSHZsUecGGZgkb", "/ip4/35.198.245.220/tcp/3333" ]
+bootstrap_address = [ "12D3KooWJNUNgs1UM9mw1Pfskxk4o3VWz37cpneSHZsUecGGZgkb", "/ip4/172.18.0.9/udp/3333/quic-v1" ]
```

Comment:
can you provide some details about this change?

Reply:
Sorry, this one is a copy-paste mistake. Thanks for catching it!